schemas/com.snowplowanalytics.snowplow/failure/jsonschema/1-0-0

```diff
@@ -0,0 +1,49 @@
+{
+    "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+    "self": {
+      "vendor": "com.snowplowanalytics.snowplow",
+      "name": "failure",
+      "format": "jsonschema",
+      "version": "1-0-0"
+    },
+    "type": "object",
+    "properties": {
+        "source": {
+            "type": ["string", "null"],
+            "description": "Source of the error. If the failure happened during enrichment, it'll be the enriched field - for example `user_ipaddress`, `unstruct` or `contexts`"
+        },
+        "failureType": {
+            "type": "string"
```

Comment:
`enum` for safer usage, wdyt @colmsnowplow ?

Reply:
No objection in principal, but need to validate that changing the enum in future won't result in any problems.

Reply:
Enum can still be a little awkward with Redshift as I don't believe we bump the maxLength as of yet.